Unittest.py

```diff
@@ -606,7 +606,7 @@ def test_one_hint_per_goal(self):
                     # We need at least one hint per goal, but it doesn't matter if there are duplicates
                     # of a goal hint or more than one hint for the goal.
                     for hint in spoiler['gossip_stones'].values():
-                        if goals[g].lower() in hint['text'].replace('#','').lower():
+                        if goals[g].lower() in hint['text'].replace('#','').lower() or goals[g].lower() == "path of the hero":
```

Comment:
It works if this change is removed now, right? If so we should go ahead and remove it

Reply:
No, it still fails without this change.